Formula/cargo-udeps.rb

```diff
@@ -15,7 +15,8 @@ class CargoUdeps < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "40f97acc5f1d9746d17a0e4ddf080876a18823443a1575df6689bd914028ba25"
   end
 
-  depends_on "rust" => [:build, :test]
+  depends_on "rust" => :build
+  depends_on "rustup-init" => :test
```

Comment:
We should make this use Homebrew `libgit2` and `libssh2` as well. And check linkage as before (and for OpenSSL while we're at it).